packages/trader/src/Modules/Trading/Components/Elements/payout-per-point-mobile.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Icon, Money, Text, Popover } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+import { observer, useStore } from '@deriv/stores';
+import { getContractSubtype, isVanillaContract } from '@deriv/shared';
+
+const PayoutPerPointMobile = observer(() => {
+    const {
+        modules: { trade },
+    } = useStore();
```

Comment:
i dont think we have `trade` in useStore. `trade-store.js` exists in trader's package and not core. I believe we have to use `useTraderStore` directly.